lib/views/physician/physician_home.dart

```diff
@@ -53,6 +53,27 @@ class PhysicianHome extends StatelessWidget {
                   )
                 ],
               )),
+          ElevatedButton(
```

Comment:
Not sure if we want an "Edit Profile" button that is like this. Maybe we can look into creating a task in the future so that the profile is accessed and it displays what information we have, and then have a button from inside the profile view that allows for edits.